src/fugw/solvers/utils.py

```diff
@@ -703,6 +710,146 @@ def solver_mm_sparse(
     )
 
 
+def solver_mm_l2_sparse(
```

Comment:
If the function is meant to be public (not sure that it should be), please add at least a one-line docstring.

Reply:
I agree, most functions in `solvers/utils.py` are not meant to be public (because they are called through `fugw.mappings` instances, which themselves call `fugw.solvers` instances).
I added a docstring, and suggest we hide all non-public functions in a follow-up PR.